src/core/time.d

```diff
@@ -1552,19 +1552,25 @@ public:
 
         The format returned by toString may or may not change in the future.
       +/
-    string toString() const nothrow pure @safe
+    void toString(scope void delegate(in char[]) nothrow pure @safe sink)
+        const scope @safe nothrow pure
     {
-        static void appListSep(ref string res, uint pos, bool last)
+        alias SinkT = typeof(sink);
+        static immutable units = [
+            "weeks", "days", "hours", "minutes", "seconds", "msecs", "usecs"
+        ];
```

Comment:
Changed the `TT` (typetuple) to a manifest constant, as it's allocated once. Had to switch the `foreach` to `static foreach` (it was `static` anyway before) and use `return` instead of `break` because the frontend insists on labeled breaks but then doesn't see the label.

Reply:
> because the frontend insists on labeled breaks but then doesn't see the label.

Is there an issue number for this?